mpas_analysis/config.default

```diff
@@ -300,6 +300,16 @@ baseDirectory = /dir/to/seaice/reference
 # directory where ocean reference simulation results are stored
 baseDirectory = /dir/to/seaice/reference
 
+[icebergObservations]
+## options related to iceberg observations with which the results will be
+## compared
+
+# directory where sea ice observations are stored
+baseDirectory = /dir/to/seaice/observations
+concentrationSH = Altiberg/Altiberg_1991-2017.nc
```

Comment:
You could just hard code this in the file, since it's the only observations we're going to support and the task for reading them in is quite specific to this file.  Many other tasks do this.  The example you were working from doesn't, though.